openedx/core/djangoapps/coursegraph/README.rst

```diff
@@ -0,0 +1,106 @@
+
+Coursegraph Support
+-------------------
+
+This app exists to write data to "Coursegraph", a tool enabling Open edX developers and support specialists to inspect their platform instance's learning content. Coursegraph itself is simply an instance of Neo4j, which is an open-source graph database with a web interface.
+
+Deploying Coursegraph
+=====================
+
+As of the Maple Open edX release, Coursegraph is *not* automatically provisioned by the community installation, and is *not* considered a "supported" part of the platform. However, operators may find the the `neo4j Ansible playbook`_ useful as a starting point for deploying their own Coursegraph instance. Alternatively, Neo4j also maintains an official `Docker image`_.
+
+In order for Coursegraph to have queryable data, learning content from LMS must be written to Coursegraph using the ``dump_to_neo4j`` management command included in this app. In order for the data to stay up to date, it must be periodically refreshed, either manually or via an automation server such as Jenkins.
+
+**Please note**: Access to a populated Coursegraph instance confers access to all the learning content in the related Open edX LMS/CMS. The basic authentication provided by Neo4j may or may not be sufficient for your security needs. Consider taking additional security measures, such as restricting Coursegraph access to only users on a private VPN.
+
+.. _neo4j Ansible playbook: https://github.com/edx/configuration/blob/master/playbooks/neo4j.yml
+
+.. _Docker image: https://neo4j.com/developer/docker-run-neo4j/
+
+
+Coursegraph in Devstack
```

Comment:
Nit: Should this section go into devstack's README instead? Or maybe there should just be a pointer from there to here?

Reply:
Devstack's docs tend not to contain instructions for running particular services, so it felt off to put Coursegraph-specific stuff there.

I did just add a link back to the [devstack interface](https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/devstack_interface.html), though.